### PR TITLE
Deprecating presence confidence from Item.

### PR DESCRIPTION
**Pull request by phil-impinj**

### CODE REVIEW

**Comment by phil-impinj** [comment]
@natke - handing off to you.

File: src/main/java/com/impinj/itemsense/client/data/item/Item.java [hunk 1]
```diff
@@ -32,6 +32,7 @@
     private String zone;
     private String floor;
     private String facility;
+    @Deprecated
```
Comment (davidyscott) [inline src/main/java/com/impinj/itemsense/client/data/item/Item.java]: Is it not also in ItemHistory?

Comment (natke) [inline src/main/java/com/impinj/itemsense/client/data/item/Item.java]: I just checked that, and it is not

**Comment by natke** [comment]
Looks good to me.

**Pull request merged**